CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the BCI-dockerfile-generator
+
+##  Opening an Pull Request
+
+The BCI-dockerfile-generator documentation has a section on contributing
+a new container or modifying an existing container. Please check the
+chapter **Adding and modifying containers** for details.
+
+Before creating a pull request, please format your source code with
+ruff, which is installed in the poetry venv by default:
+
+```bash
+poetry run ruff format
```

Comment:
question: Can format reorder imports? This would save running an extra command, it already touches the code anyway.

Reply:
no, there's a ticket on ruff side somewhere that explains they consider "reordering" a change outside formatting, so that's why you have to run check --fix instead.